{{ cookiecutter.project_name }}/{{ cookiecutter.app_name }}/tests/test_module.py

```diff
@@ -1,22 +0,0 @@
-# -*- coding: utf-8 -*-
-"""
-Testing of the module level stuff itself
-"""
-import unittest
-
-import semantic_version
-
-
-class TestModule(unittest.TestCase):
-    """
-    Test core module features, like asserting the version
-    and making sure we are exposing our classes.
-    """
-
-    @staticmethod
-    def test_version():
-        """
-        Verify we have a valid semantic version
-        """
-        import {{ cookiecutter.app_name }}
-        semantic_version.Version({{ cookiecutter.app_name }}.__version__)
```

Comment:
Not quite, one tested the settings.py version and the other tested the app's version. Since I removed the app's version this test should go away too